### PR TITLE
pkg: use another `Median`

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5692.

### What is changed and how does it work?

```
name             old time/op    new time/op    delta
MedianFilter-40     618ns ± 8%     156ns ± 9%  -74.74%  (p=0.008 n=5+5)

name             old alloc/op   new alloc/op   delta
MedianFilter-40      104B ± 0%       80B ± 0%  -23.08%  (p=0.008 n=5+5)

name             old allocs/op  new allocs/op  delta
MedianFilter-40      2.00 ± 0%      1.00 ± 0%  -50.00%  (p=0.008 n=5+5)
```

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: pkg/movingaverage/median_filter_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import (
+	"testing"
+)
+
+func BenchmarkMedianFilter(b *testing.B) {
+	data := []float64{2, 1, 3, 4, 1, 1, 3, 3, 2, 0, 5}
+
+	mf := NewMedianFilter(10)
+	for _, n := range data {
```
Comment (lhy1024) [inline pkg/movingaverage/median_filter_test.go:25]: Maybe we need to bench both add and get

Comment (lhy1024) [reply pkg/movingaverage/median_filter_test.go:25]: In hot statistics and hot scheduler,get is more frequently than add

Comment (lhy1024) [reply pkg/movingaverage/median_filter_test.go:25]: So I plan to calculate median in every add.cc @CabinfeverB

**Review by lhy1024 — APPROVED** [review]

**Review by nolouch — APPROVED** [review]

**Pull request merged**